tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf

```diff
@@ -1,7 +1,17 @@
 Define {
-       ANALOG_PLAYBACK_PCM	'Analog Playback'
-       ANALOG_CAPTURE_PCM	'Analog Capture'
-       HDA_ANALOG_DAI_NAME      'Analog Playback and Capture'
+	ANALOG_PLAYBACK_PCM		'Analog Playback'
+	ANALOG_CAPTURE_PCM		'Analog Capture'
+	HDA_ANALOG_DAI_NAME      	'Analog Playback and Capture'
+	DEEP_BUFFER_PIPELINE_ID		15
+	DEEP_BUFFER_PCM_ID		13
```

Comment:
again why not use the same indices as IPC3/topology1

Reply:
fixed